src/Microsoft.OData.Client/Microsoft.OData.Client.csproj

```diff
@@ -145,6 +145,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />

```

Comment:
>ewtonsoft.Json [](start = 32, length = 14)

why use Newtonsoft.Json?

can we use "System.Text.Json"

Reply:
@xuzhg That was my first choice. The issue was/is, System.Text.Json requires .NET Framework 4.6.1 and the Microsoft.OData.Client project targets 4.5. I'd be curious how you went around that in #1775 since Microsoft.OData.Edm project also targets 4.5

Reply:
Resolved by using ODataMessageReader to read the response and extract the aggregate value